client/src/components/Draftail/sources/ModalWorkflowSource.js

```diff
@@ -161,10 +29,20 @@ class ModalWorkflowSource extends Component {
     this.onClose = this.onClose.bind(this);
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  getChooserConfig(entity, selectedText) {
```

Comment:
Is it worth using Typescript abstract class for this, and making this an abstract method for instance? See https://www.typescriptlang.org/docs/handbook/2/classes.html#abstract-classes-and-members

Reply:
Sounds good - will give that a go!

Reply:
Gave it a try, but `npm run test` says it isn't valid outside of a Typescript file, and if I rename the file to `ModalWorkflowSource.ts` I get failures elsewhere (`Property 'jQuery' does not exist on type 'Global & typeof globalThis'`). I'll leave that improvement for someone who knows Typescript :-)